### DISCUSSION TITLE
pyMOR Community Meeting 2023-07-11

### DISCUSSION THREAD

**Discussion by pmli** [pymor/pymor#2079] [Announcements]
The next pyMOR Community Meeting will take place Tuesday, July 11, at 14:00 CEST and 08:00 EDT, online at:

link: https://bbb.mpi-magdeburg.mpg.de/meeting/pymor
password: THaEo

Anyone interested in pyMOR can join.

Currently on the agenda are:

- [x] pyMOR School organization (@pmli)
- [x] post release: release manager for next major release (@sdrave)
- [x] post release: assign 2-3 outstanding issues to each main dev (@sdrave)
- [x] agenda for next milestone (@sdrave)
- [x] NumpyMatrixOperators with structure (@drittelhacker) 
- [x] [GitHub community standards](https://github.com/pymor/pymor/community) (@pmli)
- [x] AOB (@sdrave)

Moderator: @pmli

**Comment by lbalicki** [comment]
- post release: release manager for next major release
  - next release manager @lbalicki 
  - add to release document: Set next release date after the release is completed
- post release: assign 2-3 outstanding issues to each main dev
  -  main devs with no assigned issues go through issues and assign themselves
- pyMOR School organization
  - Berlin hotel prices are high but so are prices all over Germany
  - Will make announcements soon telling people to book soon so it does not get crazy
  - Announce location on the homepage
  - Channels for announcement: Github Discussions, NA Digest, SIAM, MoRePaS
- NumpyMatrixOperators with structure
  - this relates to pyMEPACK
  - extended and rational projection solvers for matrix equations in pyMOR
  - triangular, hermitian or other structure that we can exploit
  - for example schur decomposed form
  - these things would probably require a separate operator class
